### PR TITLE
[script.subtitles.zimukux] 0.1.5

### PR DESCRIPTION
**Pull request by pizzamx**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

Changes:
- Adaption made for Zimuku's new anti-spoofing mechanism
- Misc. bug fix and code tweaks

### CODE REVIEW

**Comment by pizzamx** [comment]
@basrieter I have resubmitted my PR, please review at your convinience.

File: script.subtitles.zimukux/.github/workflows/main.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
-name: Zip&Rlz
```
Comment (enen92) [inline script.subtitles.zimukux/.github/workflows/main.yml:1]: please keep the .github folder out of the submission

File: script.subtitles.zimukux/CHANGES [hunk 2]
```diff
@@ -1,3 +1,8 @@
+V0.1.5 (2022/4/2)
```
Comment (enen92) [inline script.subtitles.zimukux/CHANGES:1]: this is not parsed by kodi, why not using a changelog.txt?

File: script.subtitles.zimukux/tests/test_zimuku.py [hunk 3]
```diff
@@ -48,7 +48,7 @@ def setUp(self):
 
     def get_agent(self, settings):
         return zmkagnt.Zimuku_Agent(
-            'http://zimuku.org', tmp_folder, Logger(),
```
Comment (enen92) [inline script.subtitles.zimukux/tests/test_zimuku.py:51]: please keep the tests folder out of the submission too

**Review by enen92 — CHANGES_REQUESTED** [review]
see comments

File: script.subtitles.zimukux/CHANGES (hunk 2 above)
Comment (pizzamx) [reply script.subtitles.zimukux/CHANGES]: removed as this is kept for the github project

File: script.subtitles.zimukux/tests/test_zimuku.py (hunk 3 above)
Comment (pizzamx) [reply script.subtitles.zimukux/tests/test_zimuku.py:51]: removed

**Comment by pizzamx** [comment]
> see comments

@enen92 Please review changes at your convinience, thanks

**Review by enen92 — APPROVED** [review]

**Pull request merged**